lib/rules/no-else-return.js

```diff
@@ -33,7 +35,24 @@ module.exports = {
          * @returns {void}
          */
         function displayReport(node) {
-            context.report({ node, message: "Unnecessary 'else' after 'return'." });
+            context.report({
+                node,
+                message: "Unnecessary 'else' after 'return'.",
+                fix: fixer => {
+                    const sourceCode = context.getSourceCode();
+                    const startToken = sourceCode.getTokenByRangeStart(node.start);
+                    const elseToken = sourceCode.getTokenBefore(startToken);
+                    const source = sourceCode.getText(node);
+                    let fixedSource;
+
+                    if (startToken.type === "Punctuator") {
+                        fixedSource = source.substr(1, source.length - 2);
```

Comment:
This will cause an error if the `else` statement doesn't have a block:

```js
if (foo) bar()
else return baz

// gets fixed to

if (foo) bar()
return ba
```

```js
if (foo) bar()
else return

// gets fixed ti:

if (foo) bar()
retur
```

Also, it will cause an error if there are more statements on the same line as the `else` block.

```js
if (foo) { bar() }
else { return baz } qux()

// gets fixed to:

if (foo) { bar() }
return baz qux()
```